python/lsst/summit/utils/utils.py

```diff
@@ -120,7 +125,7 @@ def countPixels(maskedImage, maskPlane):
     return len(np.where(np.bitwise_and(maskedImage.mask.array, bit))[0])
 
 
-def quickSmooth(data, sigma=2):
+def quickSmooth(data: np.ndarray[int], sigma: float = 2) -> np.ndarray[int]:
```

Comment:
I think these are almost always floats not ints in the data, both input and output.